### PR TITLE
Bump up grpc-node to 1.6.7 to fix CVE-2022-25878

### PR DESCRIPTION
**Pull request by alanlvle**
https://github.com/protobufjs/protobuf.js/pull/1728

### CODE REVIEW

**Comment by wu-sheng** [comment]
What is this version bump up about?

**Comment by alanlvle** [comment]
Our international business monitoring uses skywalking-nodejs, the security scanning tool aquasec reports high-risk vulnerabilities, and dependencies need to be upgraded.

**Comment by wu-sheng** [comment]
Two things
1. Update this file according to your version bump up, https://github.com/apache/skywalking-nodejs/blob/master/dist/LICENSE#L218
2. Please make the title and description clear in the PR about which CVEs(IDs) you are going to fix.

**Review by kezhenxu94 — CHANGES_REQUESTED** [review]
Please run `npm i && npm run build`, and then include the package-lock.json into the codebase

**Comment by alanlvle** [comment]
ok

File: package-lock.json [hunk 1]
```diff
@@ -555,24 +555,24 @@
...
         "@grpc/proto-loader": "^0.6.4",
         "@types/node": ">=12.12.47"
       }
     },
     "@grpc/proto-loader": {
-      "version": "0.6.7",
-      "resolved": "https://registry.npmjs.org/@grpc/proto-loader/-/proto-loader-0.6.7.tgz",
-      "integrity": "sha512-QzTPIyJxU0u+r2qGe8VMl3j/W2ryhEvBv7hc42OjYfthSj370fUrb7na65rG6w3YLZS/fb8p89iTBobfWGDgdw==",
+      "version": "0.6.13",
+      "resolved": "https://npm.zatech.online/@grpc%2fproto-loader/-/proto-loader-0.6.13.tgz",
```
Comment (wu-sheng) [inline package-lock.json:568]: I think you set a proxy? This should be changed.

Comment (alanlvle) [reply package-lock.json:568]: OK，I update it

**Review by kezhenxu94 — APPROVED** [review]

**Pull request merged**